packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -430,20 +430,12 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
       registerBoolServiceExtension(
         name: WidgetsServiceExtensions.profileWidgetBuilds.name,
         getter: () async => debugProfileBuildsEnabled,
-        setter: (bool value) async {
-          if (debugProfileBuildsEnabled != value) {
-            debugProfileBuildsEnabled = value;
-          }
-        },
+        setter: (bool value) async => debugProfileBuildsEnabled = value,
```

Comment:
this should be a body (`{ }`) rather than a return expression (`=>`) because otherwise it'll return bool instead of null. Might not matter, but better to be pedantic about this kind of thing since otherwise people might come to depend on it.

Reply:
done